lib/fizzy/instructions.hpp

```diff
@@ -16,6 +17,24 @@ struct InstructionMetrics
     /// The stack height change caused by the instruction execution,
     /// i.e. stack height _after_ execution - stack height _before_ execution.
     int8_t stack_height_change;
+
+    /// The largest acceptable alignment value satisfying `2 ** max_align < memory_width` where
+    /// memory_width is the number of bytes the instruction operates on.
+    ///
+    /// This field may contain invalid value for instructions not needing it.
```

Comment:
It's default-initialized to 0, I wouldn't call it invalid

Reply:
0 is invalid on non-memory ops, because the change preferred by @chfast makes 0 a valid value (1<<0 = 1).

Reply:
0 is valid value for memory instructions, and nonrelevant for non-memory instructions. To me it never contains "invalid value" like garbage.

Reply:
It does contain invalid value for non-memory ops.